modules/social_features/social_profile/social_profile.module

```diff
@@ -369,6 +369,14 @@ function social_profile_preprocess_profile(array &$variables) {
     $variables['profile_hero_styled_image_url'] = ImageStyle::load('social_xx_large')
       ->buildUrl($profile->field_profile_banner_image->entity->getFileUri());
   }
+  else {
+    if ($file_uuid = $profile->field_profile_banner_image->getSetting('default_image')['uuid']) {
```

Comment:
We should move this outside of the if to avoid assignment in the if statement (it doesn't add succinctness here).

We should also check we got something back before accessing `uuid` (isset/empty?) because an empty array here could throw errors.

Reply:
I have updated it, but need fix problem with access on the imege